packages/loaders/json-schema/src/addExecutionLogicToComposer.ts

```diff
@@ -165,7 +165,7 @@ ${operationConfig.description || ''}
         }
         const requestInit: RequestInit = {
           method: httpMethod,
-          headers,
+          headers: { accept: 'application/json', ...headers },
```

Comment:
@ardatan it's a patchy fix, what do you think?

Reply:
Not sure if we should send `json` by default, but instead only for openapi we can set it to `application/json` if `contentKey` is `*/*`. What do you think?
https://github.com/Urigo/graphql-mesh/blob/762ad8e644335fa7baf2bacdb317314eb410be3e/packages/loaders/openapi/src/getJSONSchemaOptionsFromOpenAPIOptions.ts#L229

Reply:
```ts
operationConfig.headers.accept = contentKey === '*/*' ? 'application/json' : contentKey; 
```

Reply:
tried that. `contentKey` is never `'*/*'`, it's coming from somewhere else.
that's why I switched the current method